_maps/cyberiad.dm

```diff
@@ -24,7 +24,7 @@ z7 = empty
 
         #define MAP_FILE "cyberiad.dmm"
         #define MAP_NAME "NSS Cyberiad"
-        #define MAP_TRANSITION_CONFIG list(MAIN_STATION = CROSSLINKED, CENTCOMM = SELFLOOPING, TELECOMMS = CROSSLINKED, CONSTRUCTION = CROSSLINKED, MINING = CROSSLINKED, DERELICT = CROSSLINKED, EMPTY_AREA = CROSSLINKED)
+        #define MAP_TRANSITION_CONFIG list(DECLARE_LEVEL(MAIN_STATION,CROSSLINKED,list(STATION_LEVEL,STATION_CONTACT,REACHABLE,AI_OK)), DECLARE_LEVEL(CENTCOMM,SELFLOOPING,list(ADMIN_LEVEL,BLOCK_TELEPORT,IMPEDES_MAGIC)), DECLARE_LEVEL(TELECOMMS,CROSSLINKED,list(REACHABLE,BOOSTS_SIGNAL,AI_OK)), DECLARE_LEVEL(CONSTRUCTION,CROSSLINKED,list(REACHABLE)), DECLARE_LEVEL(MINING,CROSSLINKED,list(REACHABLE,STATION_CONTACT,HAS_WEATHER,ORE_LEVEL,AI_OK)), DECLARE_LEVEL(DERELICT,CROSSLINKED,list(REACHABLE)), DECLARE_LEVEL(EMPTY_AREA,CROSSLINKED,list(REACHABLE)))
```

Comment:
please add spacing to this and newlines, it's not even slightly readable like this